project.php

```diff
@@ -1003,7 +1003,7 @@ function do_early_uploads()
         echo "<input type='hidden' name='tpnv' value='1'>\n";
         echo "<b>", _("Add title page and verso from directory or zip file:"), "</b>";
         echo "<br>\n";
-        $initial_rel_source = "$user_dir/";
+        $initial_rel_source = "Users/$user_dir/";
```

Comment:
Perhaps so, but I don't know if it's worth doing. I think the only two places it's used are in this file. The only reason for updating this was so that it wouldn't be flat-outright wrong if a site chooses to enable metadata. The old version of the upload script was much less complex, and did not have separate `Users` and `Commons` directories; I suspect this just got overlooked when the script was enhanced.